git-cliff-core/src/config.rs

```diff
@@ -124,6 +124,12 @@ pub struct RemoteConfig {
 	pub github: Remote,
 }
 
+impl fmt::Display for RemoteConfig {
+	fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+		write!(f, "{}", self.github)
+	}
+}
```

Comment:
it's required here:
![image](https://github.com/orhun/git-cliff/assets/11428655/c73371bc-c4ee-4afb-9182-d1d85a427aa4)
because I had to add the `format`

Reply:
actually this is wrong and it's causing tests to fail.
This gets injected as a struct in tera, not as a string

Reply:
maybe I fixed this in [9c2f038](https://github.com/orhun/git-cliff/pull/613/commits/9c2f03850a16cd2b43131038bbc036ea0556dede)